openmc/deplete/helpers.py

```diff
@@ -175,6 +175,20 @@ def generate_tallies(self, materials, scores):
         self._rate_tally.writable = False
         self._rate_tally.scores = scores
         self._rate_tally.filters = [MaterialFilter(materials)]
+        self._rate_tally_means_cache = None
+
+    @property
+    def rate_tally_means(self):
+        # If the mean cache is empty, fill it once with this transport cycle's results
+        if self._rate_tally_means_cache is None:
+            self._rate_tally_means_cache = self._rate_tally.mean
+        return self._rate_tally_means_cache
+
+    def reset_tally_means(self):
+        """Reset the cached mean rate tallies.
+        This step must be performed after each transport cycle
```

Comment:
Minor update to adhere to the numpydoc spec
```suggestion

    .. note::
        
            This step must be performed after each transport cycle
```